warehouse/ingest-core/src/main/java/datawave/ingest/data/normalizer/SimpleGroupFieldNameParser.java

```diff
@@ -57,9 +71,55 @@ public NormalizedContentInterface extractFieldNameComponents(NormalizedContentIn
         }
     }
     
+    private String extractGroup(String[] splits) {
```

Comment:
perhaps a comment that this is returning all but the first and last split joined together.

Reply:
Sure